### PR TITLE
Fix and prettify web UI

### PR DESCRIPTION
**Pull request by ruslashev**
This adapts web UI to a change with regards to API (response is now wrapped in a `response` JSON key). Also fixes spelling, prettifies text and style, and removes things like `eval()`s. Some diffs should be viewed with whitespace change ignored.

Related to #127

### CODE REVIEW

File: tp2bmc/board/tp2bmc/overlay/mnt/var/www/js/new/m_page_disp.js [hunk 1]
```diff
@@ -1,171 +1,246 @@
...
-
-
-
+function page_changeSwitchery(element, checked) {
+  if ((element.is(':checked') && checked == false) || (!element.is(':checked') && checked == true)) {
+    element.parent().find('.switchery').trigger('click');
+  }
+}
+
+function bool2int(val) {
```
Comment (svenrademakers) [inline tp2bmc/board/tp2bmc/overlay/mnt/var/www/js/new/m_page_disp.js:7]: im not a javascript expert, but it's shocking that there is no way to cast or implicitly convert bools to integer values..

File: tp2bmc/board/tp2bmc/overlay/mnt/var/www/js/new/m_page_disp.js [hunk 2]
```diff
@@ -1,171 +1,246 @@
...
+}
+
+function page_http_req_get(uUrl, lType) {
+  $.ajax({
+    url: '' + uUrl + '',
+    type: 'GET',
+    dataType: 'text',
+    timeout: 5000,
+    cache: false,
+    beforeSend: LoadFunction,
```
Comment (svenrademakers) [inline tp2bmc/board/tp2bmc/overlay/mnt/var/www/js/new/m_page_disp.js:56]:
```suggestion

```
i guess we can remove this callback if we don't use it

File: tp2bmc/board/tp2bmc/overlay/mnt/var/www/js/new/m_page_disp.js (hunk 1 above)
Comment (ruslashev) [reply tp2bmc/board/tp2bmc/overlay/mnt/var/www/js/new/m_page_disp.js:7]: To be frank, I am not sure the original author is a javascript expert either

File: tp2bmc/board/tp2bmc/overlay/mnt/var/www/js/new/m_page_disp.js (hunk 2 above)
Comment (ruslashev) [reply tp2bmc/board/tp2bmc/overlay/mnt/var/www/js/new/m_page_disp.js]: Done

**Review by svenrademakers — APPROVED** [review]

**Pull request merged**